Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -0,0 +1,104 @@
+# Creates a pull request to boneyard a formula.
+#
+# Usage: brew boneyard-formula-pr [options...] <formula-name>
+#
+# Options:
+#   --dry-run:  Print what would be done rather than doing it.
+
+require "formula"
+require "utils/json"
+require "fileutils"
+
+module Homebrew
+  def boneyard_formula_pr
+    formula = ARGV.formulae.first
+    odie "No formula found!" unless formula
+
+    formula_path = formula.path
+    core = formula_path.to_s.include? "homebrew-core"
+    if ARGV.dry_run?
+      ohai "brew update"
+      ohai "brew tap homebrew/boneyard"
+      ohai "cp #{formula_path} #{HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-boneyard/"
+      ohai "git rm #{formula_path}"
+      ohai "Adding #{formula.name} to tab migrations" if core
+    else
+      safe_system "brew", "update"
+      safe_system "brew", "tap", "homebrew/boneyard"
+      cp formula.path, HOMEBREW_LIBRARY/"Taps/homebrew/homebrew-boneyard/"
```

Comment:
You should use `tap = Tap.fetch("homebrew", "boneyard"); tap.path` for this kind of thing.